python/cuda_cccl/pyproject.toml

```diff
@@ -19,11 +19,12 @@ requires-python = ">=3.9"
 dependencies = [
   "numba>=0.60.0",
   "numpy",
-  "cuda-python==12.9.0",
+  "cuda-bindings>=12.9.1,<13.0.0",
```

Comment:
We're still working on the `12.9.1` release.

Do you think `cuda-bindings~=12` could work here?

Reply:
Does any version of `cuda-bindings` work with CUDA 13 driver? If so, I can use that.

Reply:
12.9.1 is the latest version that works with CUDA 13, I think this lower bound is appropriate